src/senaite/core/setuphandlers.py

```diff
@@ -250,6 +260,26 @@ def add_senaite_setup_items(portal):
     setup = api.get_senaite_setup()
     add_dexterity_items(setup, items)
 
+    # disable snapshots for setup folders
+    objects = [setup.get(item[0]) for item in items]
+    map(disable_snapshots, objects)
+
+
+def disable_snapshots(obj):
+    """Disable and removes snapshots from the given object
+    """
+    # do not take more snapshots
+    alsoProvides(obj, IDoNotSupportSnapshots)
+
+    # do not display audit log
+    noLongerProvides(obj, IAuditable)
+
+    # remove all snapshots
+    annotation = IAnnotations(obj)
+    storage = annotation.get(snapshot.SNAPSHOT_STORAGE)
+    if storage:
+        del(annotation[snapshot.SNAPSHOT_STORAGE])
```

Comment:
Please create a function in the snapshot API to remove this.

Reply:
Done with https://github.com/senaite/senaite.core/pull/2479/commits/94ff2a92c1be84c8c433bad3e53216c150044de4